java/life/qbic/portal/portlet/SampleTrackingUpdatePortlet.java

```diff
@@ -373,80 +372,78 @@ public void buttonClick(Button.ClickEvent event) {
 
                   String result = EntityUtils.toString(response.getEntity());
 
+                  //TODO: remove System.out.prinln() statements
                   System.out.println("++++++++++++++++++++++++++++++++++++++++++++");
```

Comment:
Please use the logger in all cases. System println statements tend to be forgotten (even with TODO statements) and pollute our log files on production.

Reply:
This is the old code -> java